frontend/src/components/ProjectTypeDashboardCard.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import { IconProp } from '@fortawesome/fontawesome-svg-core'
+import clsx from 'clsx'
+import { FC } from 'react'
+import SecondaryCard from 'components/SecondaryCard'
+
+const ProjectTypeDashboardCard: FC<{
+  type: string
+  count: number
+  icon: IconProp
+}> = ({ type, count, icon }) => {
+  let title = 'Healthy Projects'
+  if (type === 'unhealthy') {
+    title = 'Unhealthy Projects'
+  } else if (type === 'needsAttention') {
+    title = 'Projects Needing Attention'
+  }
+
+  return (
+    <SecondaryCard
+      title={title}
+      icon={icon}
+      className={clsx('overflow-hidden transition-colors duration-300', {
+        'bg-green-100 text-green-800 hover:bg-green-200 hover:text-green-800 dark:bg-green-800 dark:text-green-400 dark:hover:bg-green-700':
```

Comment:
Can we make this blue hover in dark theme a bit lighter shade? I feel like that's too conrasting for our color scheme:
<img width="560" height="215" alt="Screenshot 2025-07-10 at 9 04 01 PM" src="https://github.com/user-attachments/assets/6ff9452e-8bde-4e98-a40d-6e67f3d6fd35" />